nubis/bin/example-ldap2consul.conf

```diff
@@ -0,0 +1,13 @@
+[ldap]
+server = ldap.domain.com
+binduser = somebinduser
+bindpass = agoodpass
+searchbase = dc=domain
+searchfilter = (cn=group)
```

Comment:
You are assuming that everyone who would deploy this tool knows LDAP search parlance. I think the idea here is that we need to make this tool easy to consume for other folks. As an example, I have no idea how LDAP search works, on the other hand passing a group name is quite simple.

I suppose the alternative would be to list concrete examples of script execution in the documentation, however that feels like more work and a lower quality user experience to me.